pyproject.toml

```diff
@@ -32,42 +28,26 @@ repository = "https://github.com/PaulSchweizer/flowpipe"
 documentation = "https://flowpipe.readthedocs.io/en/latest/"
 classifiers = [
     "Programming Language :: Python",
-    "Programming Language :: Python :: 2.6",
-    "Programming Language :: Python :: 2.7",
-    "Programming Language :: Python :: 3.4",
-    "Programming Language :: Python :: 3.5",
-    "Programming Language :: Python :: 3.6",
     "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
 ]
 
 [tool.poetry.dependencies]
-python = "~2.7 || ^3.7"
+python = "^3.10"
```

Comment:
In the classifiers we list python 3.7 - 3.10, but  this sets 3.10 as minimum compatible version. I'm not aware of any 3.10-exclusive features being used, so we should probably stick to python >= 3.8, which are the officially supported versions.

Reply:
Good call, I adjusted this

Reply:
I had to move to 3.9 because of install and compatibility problems with numpy in py3.12 and I don't want to make it too complex with defining different lib versions per python version, unless we really need the 3.8 support

Reply:
That seems fine.